### DISCUSSION TITLE
<Why I have not received any email about my student development program access?>

### DISCUSSION THREAD

**Discussion by shanshamas** [community/community#45534] [GitHub Education]
### Select Topic Area

Question

### Body

I am a student of university of education, Pakistan. I applied for student development program on 25 January. They told me that your request will be processed in one day but I still have not received any e-mail or configuration message from there side.

Does anyone have any idea why I am not getting any update from Github team????

And Why I still do not have access to the student developer pack, All of my credentials were cleared from their side???

Kindly let me know as soon as possible, I will truly appreciate the help.

Thanks.

**Comment by samuel-ma** [comment]
@shanshamas 
Don't worry, it usually takes a period of about 2-3 days to receive your student pack
It should arrive in a couple of hours 
It's clearly stated [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

**Reply by shanshamas** [reply]
thanks for the info
